book/quick-brick/native-project-dev.md

```diff
@@ -0,0 +1,132 @@
+    ```
+
+    *Notes:*
+    * The `app_version_id` is the app version id string value from earlier stage.
+    * If you prefer to run `pod update` separately, you can remove `-pu` which will execute `pod update` as part of the zapptool build process.
+
+4. Open `ZappiOS.xcworkspace` or `ZappTvOS.xcworkspace` according to earlier section.
+5. Build and run on some (iPhone / Apple TV) Simulator.
+
+### Extra information for iOS/tvOS Quick-Brick
+
+We have created extra documentations about the following issues:
+
+* [How to install a debug build on AppleTV](/quick-brick/tvOS/Environment/DebugBuildIntegration.md) - This Document will explain how to download and setup a debug version of an app on your Apple TV device using the Quickbrick SDK.
+* [How to install a release build on AppleTV](/quick-brick/tvOS/Environment/DebugBuildIntegration.md) - This Document will explain how to download and setup a release version of an app on your Apple TV device using the Quickbrick SDK.
+
+***
+
+## Android and Android T.V {#android_qb}
+
+To prepare a project for Android or Android T.V with quick-brick you need to `git clone` the [Zapp-Android](https://github.com/applicaster/Zapp-Android) repository:
+
+**Important** - The [Zapp-Android](https://github.com/applicaster/Zapp-Android) repository readme file contains information about the setup prerequisites. Please make sure you're aligned.
+
+### Create an app for Android and Android T.V with Quick-Brick Q.A using Android Emulators
+
+The following steps will guide you how to create a quick-brick app that will utilize the react-native quick-brick bundle from the CDN.
+
+1. Choose a Zapp app from the [Zapp](https://zapp.applicaster.com/) website, copy the app version id and save it for later stage.
+2. Go to the `Zapp-Android` project and run the following command:
+
+    ```base
+    rbenv local 2.5.1
+    sudo gem install bundler -v 1.17.3
+    bundle install
+    bundle exec rake prepare_workspace VERSION=<Applicastion_ID>
+    ```
+
+3. Open the Android Studio project and run the project on an Android Emulator. Note that you have two option to test the quick-brick bundle from localhost:
+
+    * Add the following argument `-PREACT_NATIVE_PACKAGER_ROOT=localhost:8081` with every gradle build command. For example:
+
+        ```base
+        <your_android_project_path>/.gradlew assembleDebug -PREACT_NATIVE_PACKAGER_ROOT=localhost:8081
+        ```
+
+        To run commands gradle command directly from Android Studio, you can add the argument by going to `Preferences > Build, Execution, Development > Compiler > Command-line Options` and add `-PREACT_NATIVE_PACKAGER_ROOT=localhost:8081`.
+
+    * Before Installing the app, open he Android Studio emulator and run the following command from your command line:
+
+        ```base
+        adb reverse tcp:8081 tcp:8081
+        ```
+
+        When your device or emulator is trying to access local port 8081, that request will be routed to your laptop’s port 8081.
```

Comment:
[...] is trying to access local port 8081, to reach the react-native packager, [ ...]